CONTRIBUTING.md

```diff
@@ -9,12 +9,13 @@ Naming a branch depednds on the changes you are working on and the type of chang
 
 1. Following branch naming patterns are recommended:
 
-| Type | Pattern | Example |
-|------|---------|---------|
-| New features | `<author-name>/feature/<feature-name>_issue<issue-id>` | `devpal/feature/intorduce-storage_issue122` |
-| Bug Fix | `<author-name>/bug/<bug-name>_issue<issue-id>` | `devpal/bug/auth-error_issue123` |
-| Hotfix (urgent bug fixes) | `<author-name>/hotfix/<bug-name>_issue<issue-id>` | `devpal/hotfix/docusign-regression_issue124` |
-| Maintenance | `<author-name>/maintenance/<maintenance-name>_issue<issue-id>` | `devpal/maintenance/run-integration-tests-in-workflows_issue125` |
+| Type | Pattern | Example | Notes |
+|------|---------|---------|-------|
+| New features | `feature/<feature-name>` | `feature/intorduce-xyz-delivery` | Used for changes that can be categorised as new features |
+| Bug Fix | `fix/<fix-name>` | `fix/auth-error` | Used for changes that can be categorised as bug fixes |
```

Comment:
@tiholic security issues comes under bug fix ?

Reply:
added security branch convention in: 5e0ca83